language/Turkish/strings.po

```diff
@@ -243,29 +254,30 @@ msgid ""
 "%s\n"
 " removed from favorites list."
 msgstr ""
+" favori listesinden kaldırıldı."
```

Comment:
You missed the line with `"%s\n"` here – it will cause the name of the system to be missing from the message.